fontc/src/main.rs

```diff
@@ -177,12 +178,14 @@ fn add_glyph_work(
     Ok(())
 }
 
-fn do_work(work: Vec<Box<dyn Work<()>>>) -> Result<(), Error> {
+fn do_work(work: Vec<Box<dyn Work>>) -> Result<(), Error> {
     let work_units = work.len();
     debug!("{} work units to execute", work_units);
+    let root_context = Context::new_root();
+    // TODO: https://github.com/googlefonts/fontmake-rs/pull/26 style execution w/task-specific contexts
     let errors: Vec<Result<(), WorkError>> = work
         .into_par_iter()
-        .map(|work| work.exec())
+        .map(|work| work.exec(&root_context))
```

Comment:
also not about this pr, but you instead of collecting results, you could just collect the errors, something like:

```rust
let errors = work
    .into_par_iter()
    .filter_map(|work| work.exec(&root_context).err())
    .collect::<Vec<_>>();
```

Reply:
...good change until then though, updating

Reply:
Done